test/Test/Hspec/LeanCheck.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE FlexibleInstances, GADTs, TypeFamilies #-}
+module Test.Hspec.LeanCheck where
+
+import GHC.Stack
+import Test.Hspec.Core.Spec
+import Test.LeanCheck.Core
+
+data Property where
+  Property :: Testable prop => prop -> Property
+
+prop :: (HasCallStack, Testable prop) => String -> prop -> Spec
+prop s = it s . Property
+
+data ShouldBe where
+  ShouldBe :: (Eq a, Show a) => a -> a -> ShouldBe
+
+infix 1 `shouldBe`
+shouldBe :: (Eq a, Show a) => a -> a -> ShouldBe
+shouldBe = ShouldBe
```

Comment:
This is unfortunately necessary to print the expected/actual values with LeanCheck.